app/controllers/quotes_controller.rb

```diff
@@ -0,0 +1,28 @@
+class QuotesController < ApplicationController
+  def show
+    @quote = Quote.find params[:id]
```

Comment:
I would also expect it to throw an error. `find` will return `nil` if no record is found, whereas `find!` will throw `ActiveRecord::RecordNotFound`. Rails will rescue this exception with a 404 page by default.